benefits/core/migrations/0002_data.py

```diff
@@ -337,9 +366,10 @@ def load_data(app, *args, **kwargs):
         index_template="core/index--sbmtd.html",
         eligibility_index_template="eligibility/index--sbmtd.html",
         enrollment_success_template="enrollment/success--sbmtd.html",
+        help_template="core/includes/help--sbmtd.html",
     )
     sbmtd_agency.eligibility_types.set([sbmtd_senior_type])
-    sbmtd_agency.eligibility_verifiers.set([sbmtd_senior_verifier])
+    sbmtd_agency.eligibility_verifiers.set([sbmtd_senior_verifier, sbmtd_mobility_pass_verifier])
```

Comment:
Oh interesting! I'm pretty sure this is the problem here:

`sbmtd_mobility_pass_verifier` was added to the agency as an eligibility verifier, but `sbmtd_mobility_pass_type` was not added to the agency's supported types.

But I'm surprised this didn't cause the Benefits app to crash, interesting... We should probably guard for this case where the `type` we are sending is empty, and throw an error.

Reply:
https://github.com/cal-itp/benefits/pull/1771/commits/e209a18fc2a53f81966933e0e7adb42a087cdd1a